lib/runtime/storage/storagediff_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2024 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package storage
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
Do you think it could be beneficial to have further tests for the child tree operations as well as the `applyToTrie` function?

Reply:
Done